[2024-09-15 17:00:49,074] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: dag_clima.cargar_data manual__2024-09-15T17:00:43.942041+00:00 [queued]>
[2024-09-15 17:00:49,089] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: dag_clima.cargar_data manual__2024-09-15T17:00:43.942041+00:00 [queued]>
[2024-09-15 17:00:49,090] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2024-09-15 17:00:49,091] {taskinstance.py:1377} INFO - Starting attempt 1 of 1
[2024-09-15 17:00:49,092] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2024-09-15 17:00:49,114] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): cargar_data> on 2024-09-15 17:00:43.942041+00:00
[2024-09-15 17:00:49,121] {standard_task_runner.py:52} INFO - Started process 1231 to run task
[2024-09-15 17:00:49,125] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'dag_clima', 'cargar_data', 'manual__2024-09-15T17:00:43.942041+00:00', '--job-id', '31', '--raw', '--subdir', 'DAGS_FOLDER/dag_clima.py', '--cfg-path', '/tmp/tmpwfnilf_s', '--error-file', '/tmp/tmpr2lnxapr']
[2024-09-15 17:00:49,127] {standard_task_runner.py:80} INFO - Job 31: Subtask cargar_data
[2024-09-15 17:00:49,224] {task_command.py:371} INFO - Running <TaskInstance: dag_clima.cargar_data manual__2024-09-15T17:00:43.942041+00:00 [running]> on host 4abe27884306
[2024-09-15 17:00:49,312] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=EmanuelGomez
AIRFLOW_CTX_DAG_ID=dag_clima
AIRFLOW_CTX_TASK_ID=cargar_data
AIRFLOW_CTX_EXECUTION_DATE=2024-09-15T17:00:43.942041+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2024-09-15T17:00:43.942041+00:00
[2024-09-15 17:00:50,829] {logging_mixin.py:115} INFO - Conexi√≥n exitosa a Redshift
[2024-09-15 17:00:50,852] {logging_mixin.py:115} INFO - {"time":{"0":"2024-09-15","1":"2024-09-16","2":"2024-09-17","3":"2024-09-18","4":"2024-09-19","5":"2024-09-20","6":"2024-09-21"},"temperature_max":{"0":17.38,"1":21.43,"2":21.58,"3":22.32,"4":21.59,"5":20.25,"6":21.1},"temperature_min":{"0":11.2,"1":11.73,"2":15.56,"3":15.83,"4":16.39,"5":13.61,"6":14.66},"precipitation":{"0":0.0,"1":0.0,"2":0.0,"3":0.0,"4":18.3,"5":0.0,"6":0.0},"sede1":{"0":"Devoto","1":"Devoto","2":"Devoto","3":"Devoto","4":"Almagro","5":"Devoto","6":"Devoto"},"sede2":{"0":"Floresta","1":"Floresta","2":"Floresta","3":"Floresta","4":"Caballito","5":"Floresta","6":"Floresta"}}
[2024-09-15 17:00:50,853] {taskinstance.py:1909} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 171, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 189, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/src/cargar_data.py", line 33, in cargar_data
    df = pd.DataFrame.from_dict(xcom_value)
  File "/home/airflow/.local/lib/python3.7/site-packages/pandas/core/frame.py", line 1593, in from_dict
    return cls(data, index=index, columns=columns, dtype=dtype)
  File "/home/airflow/.local/lib/python3.7/site-packages/pandas/core/frame.py", line 730, in __init__
    raise ValueError("DataFrame constructor not properly called!")
ValueError: DataFrame constructor not properly called!
[2024-09-15 17:00:50,878] {taskinstance.py:1420} INFO - Marking task as FAILED. dag_id=dag_clima, task_id=cargar_data, execution_date=20240915T170043, start_date=20240915T170049, end_date=20240915T170050
[2024-09-15 17:00:50,894] {standard_task_runner.py:97} ERROR - Failed to execute job 31 for task cargar_data (DataFrame constructor not properly called!; 1231)
[2024-09-15 17:00:50,910] {local_task_job.py:156} INFO - Task exited with return code 1
[2024-09-15 17:00:50,952] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
